doc/sphinx/practical-tools/utilities.rst

```diff
@@ -262,6 +268,39 @@ The following makefile rules can be extended.
   One can append lines to the generated ``.merlin`` file extending this
   target.
 
+.. _coqmakefilelocallate:
+
+CoqMakefile.local-late
+++++++++++++++++++++++
+
+The optional file ``CoqMakefile.local-late`` is included at the end of the generated
+file ``CoqMakefile``.  The following is a partial list of accessible variables:
+
+:COQ_VERSION:
```

Comment:
This variable is also available in the non `-late` file (I'm using it already). Please move it upward.
Ditto for the the next one.

Reply:
All variables are available for use in *recipes* of rules in the non-late file, as well as in the definitions of variables assigned using `=`.  They are also available in the prerequisites of rules only if you double the `$` and enable secondary expansion.  However, these variables are not available when naming the outputs of rules, nor in top-level `ifeq`/`ifneq` statements, nor in variables assigned with `:=`.  (I presume your use-cases involve the former kind and not the latter kind?)  Do you have suggestions for making this distinction more clear in the documentation?

Reply:
Indeed. I think it is fine, your explanation could also go in the doc, since the external pointer is a bit boring to follow.

Reply:
I've added some explanation in the doc, please take a look